### PR TITLE
Adding SideButtons to the SideBar

### LINKED ISSUE #432: Adding Twitter/GitHub/Radis-App buttons to the SideBar
To add Sidebars Twitter/GitHub/maybe RADIS-app/Lab buttons

### PR DESCRIPTION
**Pull request by sonikarajesh**
This pull request is to address the issue of adding Side Buttons to the the Sidebar. It aims to redirect to Radis' GitHub,  Twitter, App.
Fixes #432

### CODE REVIEW

**Comment by erwanp** [comment]
Hello @sonikarajesh , can you share a screenshot of how it looks ?

**Comment by sonikarajesh** [comment]
At the bottom of the sidebar below the QuickSearch, the links to the respective GitHub, Twitter and Radis-App is visible. 
I'm still working on why the button icons aren't showing. It's probably because the image src path ain't right. 

sidebarlink

**Comment by erwanp** [comment]
Looks good ! Once the images appear you can mark it ready-for-review and we'll merge

**Comment by erwanp** [comment]
There are no icons for [RADIS-Lab](https://radis.github.io/radis-lab/) but if you can come up with one (or use the 🔬  emoji ?) , we could add it in the links too.

**Comment by sonikarajesh** [comment]
Is this fine? 
side

File: docs/_templates/buttons.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        margin-right: 5px;
+        margin-top: 15px;
+    }
+</style>
+
+<br/>
+<p>
+<a href="https://github.com/radis/radis">
+    <img class="circle-icon"
+         src="/home/sonika/radis/docs/_static/assets/general/button_github.svg"
```
Comment (gagan-aryan) [inline docs/_templates/buttons.html:15]: It would be better to use relative file paths here, so that the build runs on all machines

**Review by gagan-aryan — CHANGES_REQUESTED** [review]

File: docs/_templates/buttons.html (hunk 1 above)
Comment (gagan-aryan) [reply docs/_templates/buttons.html:15]: Also, @anandxkumar the build error in this PR is unrelated to this PR, right ?

Comment (sonikarajesh) [reply docs/_templates/buttons.html]: I have updated and used relative file paths

**Comment by erwanp** [comment]
Test failure is totally unrelated (connection erorr to a server); @gagan-aryan I can let you merge!

**Review by erwanp — APPROVED** [review]

**Comment by gagan-aryan** [comment]
Sure @erwanp. Thanks for contributing to RADIS @sonikarajesh !!

**Comment by erwanp** [comment]
Could we add a new SideButton with a link to the main RADIS page ? https://radis.github.io/
Icon could simply be the Radis icon ?

**Comment by sonikarajesh** [comment]
> Could we add a new SideButton with a link to the main RADIS page ? https://radis.github.io/ Icon could simply be the Radis icon ?

 I have added them.
#475

**Pull request merged**